codeship-steps.yml

```diff
@@ -7,4 +7,5 @@
     #-  command: ./.ci/travis/travis.sh site
     #-  command: ./.ci/travis/travis.sh nondex
     -  command: ls -la
-    -  command: .ci/run-link-check-plugin.sh
+    -  command: SKIP_FILES=".github|appveyor.yml|circle.yml|distelli-manifest.yml|fast-forward-merge.sh|LICENSE|LICENSE.apache20|README.md|release.sh|RIGHTS.antlr|shippable.yml|wercker.yml"
+    -  command: if [[ $(git diff --name-only HEAD HEAD~1 | grep -vE "$SKIP_FILES" | cat | wc -c) > 0 ]]; then .ci/run-link-check-plugin.sh; fi
```

Comment:
please wrap long lines to be less then 100 symbols in length.

Reply:
Sure

Reply:
Done.